README.md
## Welcome to GitHub Pages

You can use the [editor on GitHub](https://github.com/tpatox10/website/edit/master/README.md) to maintain and preview the content for your website in Markdown files.

Whenever you commit to this repository, GitHub Pages will run [Jekyll](https://jekyllrb.com/) to rebuild the pages in your site, from the content in your Markdown files.

### Markdown

Markdown is a lightweight and easy-to-use syntax for styling your writing. It includes conventions for

```markdown
Syntax highlighted code block

# Header 1
## Header 2
### Header 3

- Bulleted
- List

1. Numbered
2. List

**Bold** and _Italic_ and `Code` text

[Link](url) and ![Image](src)
```

For more details see [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/).

### Jekyll Themes

Your Pages site will use the layout and styles from the Jekyll theme you have selected in your [repository settings](https://github.com/tpatox10/website/settings). The name of this theme is saved in the Jekyll `_config.yml` configuration file.

### Support or Contact

Having trouble with Pages? Check out our [documentation](https://help.github.com/categories/github-pages-basics/) or [contact support](https://github.com/contact) and we’ll help you sort it out.


php/news.txt
Update!!..... Welcome to Gasabo Information center......Murakazaneza kurubuga rwacu-------------MTN

Update..MTN...Welcome to MTN Rwanda!! for new promotions vist our service centers or call customer care line on 100 for more info....

Gasabo.php
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">  
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1"> 
  <title>Gasabo.com-Welcome</title>
 <link rel="stylesheet" href="css/bootstrap.min.css">  
 <link rel="stylesheet" href="css/style.css">  
   <!-- <script scr="js/scripts.js"></script>  --> 
</head>
<body> 

<nav id="header-nav"  class="navbar navbar-default">
  <div class="container-fluid">
    <!-- Brand and toggle get grouped for better mobile display -->
<a class="navbar-brand" href="Gasabo.php">Logo</a>
 <div id="Logo-img" alt="Logo image"></div>
 
  <span class="title">Information Center</span>


   <form class="navbar-form navbar-right" role="search">
  <div class="form-group">
    <input type="text" class="form-control" placeholder="Search">
  </div>

  <button type="submit" class="btn btn-default">Search </button>
</form>
  </div><!-- /.container-fluid -->
</nav>

<!-- <h1 id="update">Updates</h1> -->

<!-- <div class="Broadcast">
<marquee align="left" truespeed="10">
<?php require("php/index.php"); ?>
</marquee>
</div>
 -->


<nav class="navbar navbar-default">
  <div class="container-fluid">
    <!-- Brand and toggle get grouped for better mobile display -->
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
    </div>
    </div>
  


    <!-- Collect the nav links, forms, and other content for toggling -->
    <div>
    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
      <ul class="nav navbar-nav">
       <li id="navlist"><a id="home1" href="Gasabo.php">Home <span class="sr-only">(current)</span></a></li>
        <li class="dropdown" id="navlist">
       <a id="home1" href="!#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false" >About Us <span class="caret"></span></a>
          <ul class="dropdown-menu">
          <li><a href="AboutUs.html">About Us </a></li>
            <li><a href="#">Education</a></li>
            <li><a href="#">Education</a></li>
            <li><a href="#">Did you know</a></li>
            <li><a href="#">Online Market</a></li>
             <li><a href="#">*****</a></li>
            <li role="separator" class="divider"></li>
            <li><a href="#">Separated link</a></li>
            <!-- <li role="separator" class="divider"></li>
            <li><a href="#">One more separated link</a></li> -->
              <li><a href="#">Acount</a></li>
          </ul>
        </li>
        <li class="dropdown" id="navlist">
          <a id="home1" href="Categories.html" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Areas <span class="caret"></span></a>
          <ul class="dropdown-menu">
            <li><a href="#">Schools</a></li>
            <li><a href="#">Companies</a></li>
              <li><a href="#">Hotels</a></li>
                <li><a href="#">Hospitals</a></li>
                   <li><a href="#">Phamacies</a></li>
            <li role="separator" class="divider"></li>
            <li><a href="#">Insurence</a></li>
             <li><a href="www.Sonarwa.com">Sonarwa</a></li>
          <li role="separator" class="divider"></li>  
         
             <li><a href="#">Banks</a></li>
             <li><a href="#">Super Markets</a></li>
             <li><a href="#">Churches</a></li>
              <li><a href="#">Acount</a></li>
          </ul>

        </li>&nbsp;
        <li class="dropdown" id="navlist">
          <a id="home1" href="Partners.html" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Partners <span class="caret"></span></a>
          <ul class="dropdown-menu">
            <li><a href="http://www.mtn.co.rw/">MTN</a></li>
             <li><a href="http://www.tigo.co.rw/">Tigo</a></li>
             <li><a href="#">K-Lab</a></li>
             <li><a href="#">RDB</a></li>
              <li><a href="#">PLP</a></li>
              <li><a href="#">LDK</a></li>
              <li><a href="#">Carla Arthur</a></li>
              <li><a href="#">Bank of Kigali</a></li>

          </ul> 
        </li>

        <li class="dropdown" id="navlist">
          <a id="home1" href="Entertainment.html" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Entertainment <span class="caret"></span></a>
          <ul class="dropdown-menu">
            <li><a href="#">Rising stars</a></li>
             <li><a href="#">Jokes</a></li>
             <li><a href="#">Miscellaneous</a></li>
              
          </ul>
        </li>
          <li class="dropdown" id="navlist">
          <a id="home1" href="Categories.html" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Education <span class="caret"></span></a>
          <ul class="dropdown-menu">
            <li><a href="#">Did you know</a></li>
            <li><a href="#">Umuco&Nururimi</a></li>
            <li role="separator" class="divider"></li>
            <li><a href="#">SAT/TOEL</a></li>
          <li role="separator" class="divider"></li>  
            <li><a href="#">Burse</a></li>
             <li><a href="#">$$$$$$$$</a></li>
             
          </ul>

        </li>&nbsp;

            <!--  <li id="navlist"><a id="home1" href="pages/New deals.html">New deals <span class="sr-only">(current)</span></a></li> &nbsp; -->

              <li id="navlist"><a id="home1" href="web/index.html">Sell online <span class="sr-only">(current)</span></a></li> &nbsp;

               <li id="navlist"><a id="home1" href="Made in Rwanda.html">Made in Rwanda <span class="sr-only">(current)</span></a></li> &nbsp;
                <li id="navlist"><a id="home1" href="Contact.html">Contact <span class="sr-only">(current)</span></a></li>&nbsp;

                 <li id="navlist"><a id="home1" href="Create acount.html">Create acount <span class="sr-only">(current)</span></a></li>&nbsp; 
                  <li id="navlist"><a id="home1" href="Sign in.html">Sign in <span class="sr-only">(current)</span></a></li>
      </ul>
    </div><!-- /.navbar-collapse -->

  </div><!-- /.container-fluid -->

</nav>
<!-- <h1 id="update">Updates</h1> -->

<div class="Broadcast">
<marquee align="left" truespeed="10">
<?php require("php/index.php"); ?>
</marquee>
</div>

<div>
  <aside>
<h2>Advert area</h2>
</aside>
</div>

<div id="main-content" class="container">


  <article class="main">
    
    <!--  <img src="http://placehold.it/400x300" alt="">  -->
    <div id="home-image">
     <img src="images/image.jpg" alt=" images"> 
      
    </div>
  

  <p>This website gives you the most and recent information about the market. You can get information about different area such as Telecommunication,banking, schools,supermarkets and more others. It also provides online buying and selling services and advertisiment as well.  </p>

   <p>This website gives you the most and recent information about the market. You can get information about different area such as Telecommunication,banking, schools,supermarkets and more others. It also provides online buying and selling services and advertisiment as well.  </p>

    <p>This website gives you the most and recent information about the market. You can get information about different area such as Telecommunication,banking, schools,supermarkets and more others. It also provides online buying and selling services and advertisiment as well.  </p>

  
  </article>
  </div>
  <span class="glyphicons glyphicons-call-record"></span>
  

<footer class="panel-footer">
<div class="container">
<div class="row">
<section id="Address" class="col-sm-4">
<span>Address</span><br>
1st Address: Nyarugenge-Kigali
                   Rwanda
2nd Address: Ashesi University
            Accra-Ghana
<hr class="visible-xs"> 
</section><br>

<section id="Contact" class="col-sm-4">
<span>Contact</span><br>

P.o.Box; 0000 Kigali<br>
Tel: +250783311517 / +233505837818

     <br>

E-mail: tpatox10@gmail.com
<hr class="visible-xs"> 
</section><br>

<section id=" Customer Support" class="col-sm-4">
<span>Customer Support</span><br>

Toll Free: 100<br>
Mobile: +250728311517<br>
E-mail: tpatox100@gmail.com
<hr class="visible-xs"> 
</section><br>
  

</div>
  


</div><br>




  <div class="text-center"> &copy; 2017 &nbsp; Personal website for getting useful Information &nbsp;<span class="separator">|</span> 
  Designed by Ndabarasa Patrick <a href="http://www.Gasabo.com">Click here</a>

  </div>

</footer><br>


<!-- jQuery (Bootstrap js plugins depends on it) -->
 <script src="js/jquery-3.1.1.min.js"></script>
 <script src="js/bootstrap.min.js"></script> 
 <script src="js/script.js"></script>
</body>
</html>

Entertainment.php
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">  
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1"> 
	<title>Gasabo.com-Welcome</title>
 <link rel="stylesheet" href="css/bootstrap.min.css">  
 <link rel="stylesheet" href="css/style.css">  
	 <!-- <script scr="js/scripts.js"></script>  --> 
</head>
<body> 

<nav id="header-nav"  class="navbar navbar-default">
  <div class="container-fluid">
    <!-- Brand and toggle get grouped for better mobile display -->
<a class="navbar-brand" href="Gasabo.html">Logo</a>
 <div id="Logo-img" alt="Logo image"></div>
 <!-- <title>Information Center</title>
 <span>Information Center</span> -->
  <a class="link-title" >Information Center</a> 


   <form class="navbar-form navbar-right" role="search">
  <div class="form-group">
    <input type="text" class="form-control" placeholder="Search">
  </div>

  <button type="submit" class="btn btn-default">Search </button>
</form>
  </div><!-- /.container-fluid -->
</nav>


<nav class="menu navbar-default">
  <div class="container-fluid">
    <!-- Brand and toggle get grouped for better mobile display -->
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
    </div>
    </div>
  


    <!-- Collect the nav links, forms, and other content for toggling -->
    <div>
    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
      <ul class="nav navbar-nav">
       <li ><a id="home1" href="Gasabo.html">Home <span class="sr-only">(current)</span></a></li>
    
             <li ><a id="home1" href="New deals.html">New deals <span class="sr-only">(current)</span></a></li> &nbsp; 
                <li ><a id="home1" href="Contact.html">Contact <span class="sr-only">(current)</span></a></li>&nbsp;
                 <li ><a id="home1" href="Create acount.html">Create acount <span class="sr-only">(current)</span></a></li>&nbsp; 
                  <li ><a id="home1" href="Sign in.html">Sign in <span class="sr-only">(current)</span></a></li>
      </ul>
    </div><!-- /.navbar-collapse -->

  </div><!-- /.container-fluid -->

</nav>

<h1 id="update">Updates</h1>

<div class="Broadcast">
<marquee align="left" truespeed="10">
<?php require("php/index.php"); ?>
</marquee>
</div>



  <aside>
<h2>Advert area</h2>
</aside>
</div>

<div id="main-content" class="container">


  <article class="main">
    
    <!--  <img src="http://placehold.it/400x300" alt="">  -->
    <div id="home-image">
     <img src="images/image.jpg" alt=" images"> 
      
    </div>
  

  <p>This website gives you the most and recent information about the market. You can get information about different area such as Telecommunication,banking, schools,supermarkets and more others. It also provides online buying and selling services and advertisiment as well.  </p>

   <p>This website gives you the most and recent information about the market. You can get information about different area such as Telecommunication,banking, schools,supermarkets and more others. It also provides online buying and selling services and advertisiment as well.  </p>

    <p>This website gives you the most and recent information about the market. You can get information about different area such as Telecommunication,banking, schools,supermarkets and more others. It also provides online buying and selling services and advertisiment as well.  </p>

  
  </article>
  </div>
  <span class="glyphicons glyphicons-call-record"></span>
  

<footer class="panel-footer">
<div class="container">
<div class="row">
<section id="Address" class="col-sm-4">
<span>Address</span><br>
1st Address: Nyarugenge-Kigali
                   Rwanda
2nd Address: Ashesi University
            Accra-Ghana
<hr class="visible-xs"> 
</section><br>

<section id="Contact" class="col-sm-4">
<span>Contact</span><br>

P.o.Box; 0000 Kigali<br>
Tel: +250783311517 / +233505837818

     <br>

E-mail: tpatox10@gmail.com
<hr class="visible-xs"> 
</section><br>

<section id=" Customer Support" class="col-sm-4">
<span>Customer Support</span><br>

Toll Free: 100<br>
Mobile: +250728311517<br>
E-mail: tpatox100@gmail.com
<hr class="visible-xs"> 
</section><br>
  

</div>
  


</div><br>




  <div class="text-center"> &copy; 2017 &nbsp; Personal website for getting useful Information &nbsp;<span class="separator">|</span> 
  Designed by Ndabarasa Patrick <a href="http://www.Gasabo.com">Click here</a>

  </div>

</footer><br>
<meta name="Market information" content="This website give you most and recent information on the market"></meta>
<src img="">







<!-- jQuery (Bootstrap js plugins depends on it) -->
 <script src="js/jquery-3.1.1.min.js"></script>
 <script src="js/bootstrap.min.js"></script> 
 <script src="js/script.js"></script>
</body>
</html>

Contact.php
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">  
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1"> 
  <title>Gasabo.com-Welcome</title>
 <link rel="stylesheet" href="css/bootstrap.min.css">  
 <link rel="stylesheet" href="css/style.css">  
   <!-- <script scr="js/scripts.js"></script>  --> 
</head>
<body> 

<nav id="header-nav"  class="navbar navbar-default">
  <div class="container-fluid">
    <!-- Brand and toggle get grouped for better mobile display -->
<a class="navbar-brand" href="Gasabo.php">Logo</a>
 <div id="Logo-img" alt="Logo image"></div>
 <!-- <title>Information Center</title>
 <span>Information Center</span> -->
  <a class="link-title" >Information Center</a> 


   <form class="navbar-form navbar-right" role="search">
  <div class="form-group">
    <input type="text" class="form-control" placeholder="Search">
  </div>

  <button type="submit" class="btn btn-default">Search </button>
</form>
  </div><!-- /.container-fluid -->
</nav>


<nav class="menu navbar-default">
  <div class="container-fluid">
    <!-- Brand and toggle get grouped for better mobile display -->
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
    </div>
    </div>
  


    <!-- Collect the nav links, forms, and other content for toggling -->
    <div>
    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
      <ul class="nav navbar-nav">
       <li ><a id="home1" href="Gasabo.php">Home <span class="sr-only">(current)</span></a></li>
 </nav>

		<div id="form">
		<form method="post" action="process.php">

		<p>
		<label for="name">Name:</label>
		<input type="text" name="name" placeholder="Your name">
		</p>

		<p>
		<label for="name">Email:</label>
		<input type="text" name="email" placeholder="email">
		</p>

		<p>
		<label for="name">Subject:</label>
		<input type="text" name="subject">
		</p>

		<p>
        <label for="name">Message:</label>
		<textarea id="message" name="message" id="" cols="30"  rows="10">
        </textarea>
		</p>
		<p>
			<input id="submit" type="submit" name="submit" value="Send Message">
		</p>
			

		</form>
			
		</div>
  

<footer class="panel-footer">
<div class="container">
<div class="row">
<section id="Address" class="col-sm-4">
<span>Address</span><br>
1st Address: Nyarugenge-Kigali
                   Rwanda
2nd Address: Ashesi University
            Accra-Ghana
<hr class="visible-xs"> 
</section><br>

<section id="Contact" class="col-sm-4">
<span>Contact</span><br>

P.o.Box; 0000 Kigali<br>
Tel: +250783311517 / +233505837818

     <br>

E-mail: tpatox10@gmail.com
<hr class="visible-xs"> 
</section><br>

<section id=" Customer Support" class="col-sm-4">
<span>Customer Support</span><br>

Toll Free: 100<br>
Mobile: +250728311517<br>
E-mail: tpatox100@gmail.com
<hr class="visible-xs"> 
</section><br>
  

</div>
  


</div><br>




  <div class="text-center"> &copy; 2017 &nbsp; Personal website for getting useful Information &nbsp;<span class="separator">|</span> 
  Designed by Ndabarasa Patrick <a href="http://www.Gasabo.com">Click here</a>

  </div>

</footer><br>
<meta name="Market information" content="This website give you most and recent information on the market"></meta>
<src img="">







<!-- jQuery (Bootstrap js plugins depends on it) -->
 <script src="js/jquery-3.1.1.min.js"></script>
 <script src="js/bootstrap.min.js"></script> 
 <script src="js/script.js"></script>
</body>
</html>